xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -4391,6 +4418,9 @@ bool CVideoPlayer::OnAction(const CAction &action)
       }
       else
         break;
+    case ACTION_TOGGLE_COMMSKIP:
+      ToggleCommSkip();
```

Comment:
ahh, I think it is better to place those two lines of code here instead of call a function.